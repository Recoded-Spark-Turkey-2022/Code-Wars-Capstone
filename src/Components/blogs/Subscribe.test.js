import React from "react";
import renderer from "react-test-renderer";
import SubscribeForm from './Subscribe';
import { MemoryRouter } from "react-router-dom";

it("SubscribeForm renders correctly", () => {
    const test = renderer
        .create(
            <MemoryRouter>
                <SubscribeForm/>
            </MemoryRouter>
        )
        .toJSON();
    expect(test).toMatchSnapshot();
});