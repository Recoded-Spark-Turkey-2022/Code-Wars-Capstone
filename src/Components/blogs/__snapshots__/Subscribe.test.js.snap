// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Blogarticle renders correctly 1`] = `
<div
  className="pt-[2rem] sm:w[5rem]   "
>
  <form>
    <label
      htmlFor="email"
    >
      <input
        className=" w-[15rem] h-[2.5rem] text-xl leading-loose text-gray-500 bg-white border-2 text-left border-gray-500 "
        id="email"
        onChange={[Function]}
        placeholder="Enter your e-mail"
        type="email"
        value=""
      />
      <button
        className="  w-[3rem] h-[2.5rem] leading-loose bg-[#2DD3E3] border-2  rounded-br-md border-gray-500 translate-y-1  hover:bg-cyan-500 hover:text-white "
        onClick={[Function]}
        type="submit"
      >
         
        <img
          alt="sendicon"
          className="  hover:img-white mx-auto w-[1.5rem]"
          src="Style.png"
        />
         
      </button>
      <div />
      <div />
    </label>
  </form>
</div>
`;
